### PR TITLE
Add personalization to the existing ML task

### PR DESCRIPTION
**Pull request by mrksr**

### CODE REVIEW

File: kafker/markov/views.py [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
+@app.table_route(table=personal_dictionary, match_info="author")
+async def generate_post(web, request, author, length=50):
     def word_generator():
         current_word = "^"
+        personal_words = personal_dictionary[author]
         while True:
             next_words = list(bigrams[current_word])
             next_weights = [
-                bigram_weights[current_word, next_word] for next_word in next_words
+                (10.0 if next_word in personal_words else 1.0)
```
Comment (apaleyes) [inline kafker/markov/views.py]: 1.0 and 10.0 should be constants, or hyperparameters. Maybe WORD_WEIGHT and PERSONAL_WORD_WEIGHT

File: kafker/markov/agents.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    async for lhs, rhs in new_bigrams:
+        bigrams[lhs].add(rhs)
+        bigram_weights[lhs, rhs] += 1
+
+
+@app.agent(personal_dictionary_rebuilds)
+async def rebuild_personal_weights(personal_dictionary_rebuilds):
+    async for author in personal_dictionary_rebuilds.group_by(
+        lambda x: x, name="by-author"
+    ):
```
Comment (apaleyes) [inline kafker/markov/agents.py]: I'd break this line in two (group by, followed by for), just for easier reading

**Review by apaleyes — APPROVED** [review]
Just some nitpicking comments!

File: kafker/markov/views.py (hunk 1 above)
Comment (mrksr) [reply kafker/markov/views.py]: Just added a constant for `PERSONAL_WORDS_WEIGHT` as the 1 will always be 1. Personal words are weighted relative to everything else.

**Pull request merged**